.github/workflows/build.yml

```diff
@@ -242,6 +242,21 @@ jobs:
       CARGO_INCREMENTAL: "0"
       SCCACHE_CACHE_SIZE: "100GB"
     steps:
+      - name: Free Disk Space (Ubuntu)
```

Comment:
why this?

Reply:
Not sure why it was needed, also part of #10. @rimbi can you give a bit of insight on this, please?